libraries/rush-lib/src/api/CommandLineConfiguration.ts

```diff
@@ -456,27 +456,28 @@ export class CommandLineConfiguration {
         buildCommandOriginalPhases = buildCommand.originalPhases;
         this.commands.set(buildCommand.name, buildCommand);
       }
+    }
```

Comment:
@dmichon-msft Just curious, what do we expect to happen if two different Rush plugins define a phase with the same phase name?

Reply:
Historically plugins weren't able to define commands, so this is new to me.